metrics/grafana/tiflash_summary.json

```diff
@@ -4186,6 +4186,22 @@
               "intervalFactor": 1,
               "legendFormat": "AIO",
               "refId": "F"
+            },
+            {
+              "expr": "sum(rate(tiflash_system_profile_event_PSV3MBlobExpansion{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m]))",
+              "format": "time_series",
+              "intervalFactor": 1,
+              "hide": true,
+              "legendFormat": "BlobExpansion",
+              "refId": "G"
+            },
+            {
+              "expr": "sum(rate(tiflash_system_profile_event_PSV3MBlobReused{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m]))",
+              "format": "time_series",
+              "intervalFactor": 1,
+              "hide": true,
+              "legendFormat": "BlobReused",
```

Comment:
I think it is better to use:
1. request_qps
2. reused_ratio = BlobReused / BlobExpansion

Reply:
done